graph/step.go

```diff
@@ -105,6 +111,11 @@ func (s *Step) Validate() error {
 			return NewSelfReferencedStepError(fmt.Sprintf("Step ID: %v is self-referenced", s.ID))
 		}
 	}
+
+	if s.Cache != "" && strings.ToLower(s.Cache) != enabled && strings.ToLower(s.Cache) != disabled {
```

Comment:
>strings.ToLower(s.Cache) != enabled && strings.ToLower(s.Cache) != disabled [](start = 21, length = 75)

is it possible to use [https://golang.org/pkg/strings/#EqualFold](https://golang.org/pkg/strings/#EqualFold)?